### PR TITLE
Set up SMS reminders in Sri Lanka for next 6 months

### PR DESCRIPTION
**Pull request by qptr**
**Story card:** [sc-10955](https://app.shortcut.com/simpledotorg/story/10955)

SMS reminders in Sri Lanka have stopped going out since April/May 2023 because notification experiments weren't set up after that. This PR sets them up for 6 months from September to February.

### CODE REVIEW

**Comment by qptr** [comment]
Need to test that the translations are happening correctly. Change locale to ta-LK, en-LK and si-LK and check that notification messages are formed correctly.

File: db/data/20230808085832_set_up_six_month_sms_reminders_sri_lanka.rb [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class SetUpSixMonthSmsRemindersSriLanka < ActiveRecord::Migration[6.1]
+  first_experiment_start_time = DateTime.parse("1 Sep 2023")
+  experiment_start_times = (0..5).map do |number_of_months|
```
Comment (olttwa) [inline db/data/20230808085832_set_up_six_month_sms_reminders_sri_lanka.rb:5]: naming refactor? s/number_of_months/n.

number_of_months.months is confusing

File: db/data/20230808085832_set_up_six_month_sms_reminders_sri_lanka.rb [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+    EXPERIMENTS_DATA.map do |experiment_data|
+      ActiveRecord::Base.transaction do
+        Experimentation::Experiment.current_patients.create!(
+          name: experiment_data[:name],
+          start_time: experiment_data[:start_time],
+          end_time: experiment_data[:end_time],
+          max_patients_per_day: 5000
+        ).tap do |experiment|
+          cascade = experiment.treatment_groups.create!(description: "sms_reminders_cascade")
```
Comment (olttwa) [inline db/data/20230808085832_set_up_six_month_sms_reminders_sri_lanka.rb:29]: Can include experiment name in description of it's TG.

Comment (olttwa) [reply db/data/20230808085832_set_up_six_month_sms_reminders_sri_lanka.rb:29]: We aren't doing this historically, but can help during debugging.

File: spec/data/set_up_six_month_sms_reminders_sri_lanka_spec.rb [hunk 3]
```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
```
Comment (olttwa) [inline spec/data/set_up_six_month_sms_reminders_sri_lanka_spec.rb:1]: the specs look good!

**Review by olttwa — APPROVED** [review]
approved by akshat & @PriyangaPKini

**Pull request merged**